conf/airframes/tudelft/rot_wing_25kg.xml

```diff
@@ -464,6 +464,7 @@
         <define name="MAX_PITCH"            value="12"/>
         <define name="MIN_PITCH"            value="-20"/>
         <define name="MAX_LAT_ACCEL"        value="5.0"/>
+        <define name="ACCEL_FWD_BX_LIM"     value="3.0"/>
```

Comment:
@Dennis-Wijngaarden took it out, he reported some strange behavior. I would doubt that is because of this line though.
We can keep it out, if needed we can re-add it.